### PR TITLE
Simplify Wkt data structure

### PR DESCRIPTION
**Pull request by michaelkirk**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
- [x] I ran cargo fmt
---

~~This is based on #109, so review that first.~~ Merged!

It's a large breaking change, so I'd appreciate getting at least a couple opinions.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -142,7 +141,7 @@ impl<T> WktFloat for T where T: WktNum + Float {}
 
 #[derive(Clone, Debug, PartialEq)]
 /// All supported WKT geometry [`types`]
-pub enum Geometry<T>
```
Comment (michaelkirk) [inline src/lib.rs:145]: The crux of the change is this and deleting the old `Wkt`.

**Review by frewsxcv — APPROVED** [review]

**Comment by michaelkirk** [comment]
This sat here a long time and I forgot about it. I've rebased! Can I get another review?

**Comment by michaelkirk** [comment]
Well, I got an approval a long time ago and it really seems like no one cares much, so I'm going to merge this before it goes stale again.

**Comment by kylebarron** [comment]
Ah the luck of you merging this right when I was working on ZM support 🥲

**Pull request merged**